content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -1,30 +1,24 @@
 
-If you're new to our strategy pages, this video walkthrough will help orient you.
+If you're new to our strategy pages, this video walkthrough (last updated Feb 2022) will help orient you.
 
 <div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/71b675b67c464b0db7670a7f5025cbbe" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-## Themes
-
-Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.** We will achieve this through several important themes:
-
-### Make the power of our features easier to find and use
+## Mission
 
-Getting the most out of our product can take time as you get up to speed with what's possible. Making it easier to learn will help teams find the value more quickly than today. Our task is to find a way to make interacting with multiple languages and repositories easy to understand.
+Our mission is to **make discovering, learning, and editing code easier, no matter how complex it is, where it's stored, or how experienced you are.**
 
-### Level up our enterprise-ready features
-
-How pricing, upgrade paths, auditing, single sign-on, access control, etc. work are essential to our customers with complex organizations. In partnership with our Cloud team, we're focused on ensuring we exceed their needs so that rolling out Sourcegraph is easy to do.
+## Themes
 
-### Enable our use cases
+Apart from the work each of our teams do for our mission, across Code Graph we're particularly focused now on improving the **discoverability and ease-of-use** around our [use cases](../index.md#use-cases). Focusing on these ensures we're prioritizing areas where people get real value out of the product, reducing time-to-value reduces the chance of people giving up on our features because they are too complicated/not intuitive, and building cross-team capabilities helps us deliver features that are more than the sum of their parts (as opposed to shipping our org chart).
 
-By orienting about our [use cases](../index.md#use-cases) we will deliver solutions that are more than the sum of their parts. Our product area is central to realizing the value of most of them, so we're focused on improving how we ship features across team boundaries to make them better and better.
+Because these org-level concepts intentionally bridge team boundaries, we are especially mindful of following our [collaboration principles](../../../departments/product-engineering/process/cross-org-team-collab.md).
```

Comment:
```suggestion
Because this theme intentionally bridges team boundaries, we are especially mindful of following our [collaboration principles](../../../departments/product-engineering/process/cross-org-team-collab.md).
```